tests/cli/test_cli_install.py

```diff
@@ -28,6 +29,10 @@ def test_pre_link_message(test_recipes_channel: None, mocker, prefix):
     assert "Lorem ipsum dolor sit amet" in stdout
 
 
+@pytest.mark.skipif(
+    context.solver == "libmamba",
+    reason="conda-libmamba-solver does not use the Resolve interface",
+)
```

Comment:
Would that need to move inside the test function?

Reply:
Depends on whether `usefixtures` works as suggested heh